solutions/59.js

```diff
@@ -19,6 +21,20 @@ const solution = (num) => {
   }
   return true;
 };
+
+const solution1 = (num) => {
+  if(num === 1 || num === 0 || num === -1){
+    return false;
+  }
+  for(let i = 2; i < Math.sqrt(num); i++){
```

Comment:
When using a specific algorithm that may not be known to general audience, it is good practice to leave a comment explaining the algorithm.  In this case, the comment can say `Trial division test for Primality: divide from 2 to the square root of the number`